.github/workflows/testing-external.yml

```
@@ -1,4 +1,4 @@
-name: External Tests
+name: Dev Tests with External Dependencies
```

```suggestion
name: Dev Tests (Live Services)
```

Trying to shorten even more